inc/utils.php

```diff
@@ -68,7 +68,7 @@ function merchantProfile()
 function merchantProfileId()
 {
     static $merchantProfileId = null;
-    $merchantProfileIdOptionKey = Mollie_WC_Plugin::PLUGIN_ID . '_merchant_profile_id';
+    $merchantProfileIdOptionKey = Mollie_WC_Plugin::PLUGIN_ID . '_cached_merchant_profile_id';
```

Comment:
You don't know from here that is will be cached. 
What's the real reason to change it?

Reply:
Well, we need to change the name of the option because it's the most effective way so that new and old users can have it with the autoload enabled, that is kind of caching the option. So in order to differentiate from the non autoloaded one I've named the new cached, but maybe it's preferable _autoloaded_merchant_profile_id? Anyways once it is created the name refers to its autoload status, so I think it's descriptive. But we can always improve the name, I'm open to suggestions.

Reply:
I do not think the name should reflect it's cached, it's an option, that's enough.
I would instead simply remove the option when the plugin is updated. That will fix the problem the right way, you have to remove the old option anyway during the update so I would suggest to go with that way instead of changing the name of the option because sounds more simple.

Reply:
In a call we agreed to change the name of the option by removing the `cached` term and invert the `merchant` and `profile`. This way we'll not produce problems if in future the option driver will change.